content/blog/2020/06/2020-06-18-terminology-update.adoc

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Terminology Update"
+tags:
+- community
+- opensource
+author: markyjackson-taulia
+description: "Terminology Updates"
+opengraph:
+  image: /images/docker/dockerJenkins_social.png
+---
+
+In 2016, the Jenkins community decided to start removing offensive terminology within the project. Offensive language was deprecated in Jenkins 2.0.
+In 2017, the project began tracking areas for correction link:https://issues.jenkins-ci.org/browse/JENKINS-42816[tracking areas for correction].
+This year, a group of core contributors continued addressing this critical link:https://www.jenkins.io/blog/2020/05/06/docker-agent-image-renaming/[work]. 
+
+The Advocacy & Outreach SIG met to discussed and prioritize the continued work. The goverence board has also met and there will be more information coming regarding removal of offsenive terminology.
+Last week we took another step towards removing offensive terminology within the project by updating previous blog posts and link:https://github.com/jenkins-infra/jenkins.io/pull/3447[removing offensive terminology].
+There is more link:https://groups.google.com/forum/#!topic/jenkinsci-dev/CLR55wMZwZ8[work to do].
+
+We continue to do this much needed work and would like to remind everyone that the Jenkins project is governed by the link:https://www.jenkins.io/project/conduct/[Code of Conduct.]
+
```

Comment:
I think it makes sense to add a Call for Action for "How to contribute?". I can help with this section

Reply:
I disagree. This blog post is to discuss the work that has already been done and noting we are still working. If this blog post makes people become passionate, they will ask to help.